### PR TITLE
Guest lock screen controls

### LINKED ISSUE #172: Song information missing from lock screen controls on guest device
Observed on Nexus 7.  Lock screen controls are present, and functional, but the song album and title is missing.

To reproduce:
1. Establish a network, add songs to the playlist, and press play.
2. Lock the guest phone.
3. Press the side button to activate the lock screen.

### PR DESCRIPTION
**Pull request by theJenix**
Fixes #172 by requesting audio focus on the guest on PlayStatusMessage, and ensuring the external control client has the current song when we receive that message.

### CODE REVIEW

**Comment by theJenix** [comment]
Additional tests to add with this pull request: we want to test audio ducking on the guest, to make sure it doesn't affect the host audio.  @twrobel3, can you add this to our test script?

**Comment by dgreenhalgh** [comment]
I like this if @reidbaker does

File: SoundStream/src/com/lastcrusade/soundstream/SoundStreamExternalControlClient.java [hunk 1]
```diff
@@ -43,7 +43,7 @@
  *  Registering the MEDIA_BUTTON handler
  *  Registering a RemoteControlClient object (for lock screen remote control on ICS +)
  * 
- * @author thejenix
+ * @author Jesse Rosalia
```
Comment (reidbaker) [inline SoundStream/src/com/lastcrusade/soundstream/SoundStreamExternalControlClient.java:46]: Are you moving the docs to your name now?

Comment (theJenix) [inline SoundStream/src/com/lastcrusade/soundstream/SoundStreamExternalControlClient.java:46]: They should all be that way…I think I just missed this one.  Did I miss others?

Comment (reidbaker) [inline SoundStream/src/com/lastcrusade/soundstream/SoundStreamExternalControlClient.java:46]: SoundStream/src/com/lastcrusade/soundstream/SoundStreamExternalControlClient.java
46: \* @author thejenix

SoundStream/src/com/lastcrusade/soundstream/audio/IDuckable.java
22: \* @author thejenix

SoundStream/src/com/lastcrusade/soundstream/net/core/ISerializable.java
28: \* @author thejenix

SoundStream/src/com/lastcrusade/soundstream/net/core/IntegerLength.java
31: \* @author thejenix

SoundStream/src/com/lastcrusade/soundstream/util/LogUtil.java
24: \* @author thejenix

SoundStream-test/src/com/lastcrusade/soundstream/net/message/FileMessage.java
28: \* @author thejenix

SoundStream-test/src/com/lastcrusade/soundstream/net/message/StringMessage.java
28: \* @author thejenix

SoundStream-test/src/com/lastcrusade/soundstream/net/wire/WireSendInputStreamTest.java
38: \* @author thejenix

SoundStream-test/src/com/lastcrusade/soundstream/util/InputBufferTest.java
28: \* @author thejenix

SoundStream-test/src/com/lastcrusade/soundstream/util/MessageTestUtil.java
27: \* @author thejenix

Comment (theJenix) [inline SoundStream/src/com/lastcrusade/soundstream/SoundStreamExternalControlClient.java:46]: Well, damn.  I'll fix those after beta.

**Pull request merged**